src/bootstrap/src/lib/types/checkbox.ts

```diff
@@ -10,7 +10,7 @@ import { FieldType } from '@ngx-formly/core';
         [indeterminate]="to.indeterminate && (model || {})[key] === undefined"
         [formControl]="formControl"
         [formlyAttributes]="field">
-      <label class="custom-control-label" [for]="id">
+      <label *ngIf="to.label && !to.hideLabel" class="custom-control-label" [for]="id">
```

Comment:
use different name since `hideLabel` is used in the main wrapper:
``` 
*ngIf="to.label && to.hideCheckboxLabel !== true"
```

Reply:
Done